src/errors/runtime/templates.js

```diff
@@ -101,11 +101,6 @@ export default {
     [RUNTIME_ERRORS.unexpectedIPCHeadPacket]:                            'Cannot create an IPC message due to an unexpected IPC head packet.',
     [RUNTIME_ERRORS.unexpectedIPCBodyPacket]:                            'Cannot create an IPC message due to an unexpected IPC body packet.',
     [RUNTIME_ERRORS.unexpectedIPCTailPacket]:                            'Cannot create an IPC message due to an unexpected IPC tail packet.',
-    [RUNTIME_ERRORS.cannotFindTypescriptConfigurationFile]:              'Unable to find the TypeScript configuration file in "{filePath}"',
```

Comment:
We use these errors in the code, should we really remove them?

https://github.com/DevExpress/testcafe/blob/47a0be182c02246a5e080a55838c8c0b68eee873/src/configuration/typescript-configuration.ts#L46
https://github.com/DevExpress/testcafe/blob/41b9f22f07c040365f58553b9bbdeb71fbd0f529/src/custom-client-scripts/client-script.ts#L90
https://github.com/DevExpress/testcafe/blob/41b9f22f07c040365f58553b9bbdeb71fbd0f529/src/custom-client-scripts/client-script.ts#L97
https://github.com/DevExpress/testcafe/blob/41b9f22f07c040365f58553b9bbdeb71fbd0f529/src/custom-client-scripts/client-script.ts#L47
https://github.com/DevExpress/testcafe/blob/41b9f22f07c040365f58553b9bbdeb71fbd0f529/src/custom-client-scripts/client-script.ts#L76

Reply:
we had duplicates for them in the templates
or I'm missing something?

Reply:
Yes, you're right, my mistake :(